src/patternfly/components/Toolbar/examples/Toolbar.md

```diff
@@ -435,12 +435,12 @@ The `.pf-m-toggle-group` controls when, and at which breakpoint, filters will be
       {{> toolbar-item-search-filter button--id="expanded-content"}}
       {{#> toolbar-group toolbar-group--modifier="pf-m-filter-group"}}
         {{#> toolbar-item}}
-          {{#> select id=(concat toolbar--id '-select-checkbox-status-expanded') select--IsCheckboxSelect="true"}}
+          {{#>select select--id=(concat toolbar--id '-select-checkbox-status-expanded') select--IsCheckboxSelect="true"}}
```

Comment:
I'll stop here :D Looks like there are a bunch more in this template missing the space...

```suggestion
          {{#> select select--id=(concat toolbar--id '-select-checkbox-status-expanded') select--IsCheckboxSelect="true"}}
```